src/keys-detective/index.ts

```diff
@@ -3,21 +3,22 @@ import { buildKeys } from '../keys-builder/build-keys';
 import { messages } from '../messages';
 import { Config } from '../types';
 import { getLogger } from '../utils/logger';
+import { filterPathByLang } from '../utils/path.utils';
 import { resolveConfig } from '../utils/resolve-config';
 
 import { compareKeysToFiles } from './compare-keys-to-files';
 import { getTranslationFilesPath } from './get-translation-files-path';
 
 export function findMissingKeys(inlineConfig: Config) {
   const logger = getLogger();
-  const config = resolveConfig(inlineConfig);
+  const config = resolveConfig({ langs:[], ...inlineConfig });
```

Comment:
The only thing is that this config property has 2 different purposes when running the find command and when running the extract.

we need to make it clear in the docs if we want to use the same property.
Also, the default config should be different based on what action you are running, otherwise, you need to override it locally like you did here but I'm not sure that's a good practice in the long run.